components/dashboard/src/projects/project-context.tsx

```diff
@@ -27,25 +26,41 @@ export const ProjectContextProvider: React.FC = ({ children }) => {
     return <ProjectContext.Provider value={ctx}>{children}</ProjectContext.Provider>;
 };
 
-export function useProjectSlugs(): { projectSlug?: string; prebuildId?: string } {
-    const projectsRouteMatch = useRouteMatch<{ projectSlug?: string; prebuildId?: string }>(
-        "/projects/:projectSlug?/:prebuildId?",
-    );
+interface ProjectInfo {
+    id: string;
+    name?: string;
+}
+
+export function useProjectInfo(): ProjectInfo | undefined {
+    const projectsRouteMatch = useRouteMatch<{ projectSlug?: string }>("/projects/:projectSlug");
```

Comment:
prebuildId is omitted because we don't use it anymore?

Reply:
yes, it wasn't used